src/Process/Pool/Logger/Message/SerializableProcess.php

```diff
@@ -116,8 +122,36 @@ public function setTypeCode(string $type_code) : SerializableProcessInterface
         return $this;
     }
 
+    public function getMemoryUsageBytes() : int
+    {
+        $this->memory_usage_bytes = memory_get_usage();
+
+        return $this->memory_usage_bytes;
+    }
+
+    public function getMemoryPeakUsageBytes() : int
+    {
+        $this->memory_peak_usage_bytes = memory_get_peak_usage();
+
+        return $this->memory_peak_usage_bytes;
```

Comment:
I get why you did it this way, but can/should we instead modify this to be

```
return memory_get_peak_usage();
```

and `jsonSerialize` to be

```
return [
    'memory_peak_usage_bytes' => $this->getMemoryPeakUsageBytes()
    ...
];
```